nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -1241,6 +1246,13 @@ in
           noCheck = true; # fsck fails on a r/o filesystem
         };
       }
+      (lib.optionalAttrs setupXbootldr {
+        "/efi" = {
+          device = "/dev/disk/by-label/${espFilesystemLabel}";
+          fsType = "vfat";
+          noCheck = true; # fsck fails on a r/o filesystem
```

Comment:
Why is it RO here?

Reply:
Arguably, this could be set by the bootloader directly, not sure if this is something the test framework has to do itself?

Reply:
> Why is it RO here?

Not sure. I just mirrored the `/boot` entry above this.

> Arguably, this could be set by the bootloader directly, not sure if this is something the test framework has to do itself?

I think it's better to do it here in one place. If `setupXbootldr` is true, the VM should mount all the boot partitions created in the disk image.